applications/matter_weather_station/child_image/mcuboot_ZDebug.conf

```diff
@@ -13,6 +13,7 @@ CONFIG_PM=n
 CONFIG_MAIN_STACK_SIZE=10240
 CONFIG_MBEDTLS_CFG_FILE="mcuboot-mbedtls-cfg.h"
 
+CONFIG_BOOT_UPGRADE_ONLY=y
```

Comment:
Enabling this now will lead to build errors. Either we block this PR with upmerge or you will remove this for now and add it later on.

Reply:
It took me a while, but i found why its **not** triggering the build issues right now. You are explicitly setting `CONFIG_NRF53_UPGRADE_NETWORK_CORE=n` in application configuration. Seems obvious but i didn't spot it at first sight. You can leave this for now.

Reply:
Great, thanks for investigating that!